### DISCUSSION TITLE
`useInterval` composable

### DISCUSSION THREAD

**Discussion by ShlomoCode** [quasarframework/quasar#17000] [Ideas / Proposals] [answered]
I think it should be added `useInterval` for `setInterval`, like [useTimeout](https://quasar.dev/vue-composables/use-timeout) for `setTimeout`

**Comment by rstoenescu** [comment]
Great idea! Wanna take a shot at it? It's mostly copy paste from the useTimeout.

**Reply by rstoenescu** [reply]
Or maybe a param to useTimeout

**Comment by rstoenescu** [comment, marked as answer]
useInterval() will be available in Quasar v2.15.1
